assets/css/src/components-public/_resets.scss

```diff
@@ -0,0 +1,104 @@
+// basic default styling
+// to allow for consistent styling between mailpoet plugin and user's theme
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
```

Comment:
The `mailpoet-public.css` file may be included even on pages where there are no subscription forms. Anyway, styles applied to elements globally may affect page elements outside of MailPoet scope. Do we need such side-effects?